management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastOutboundGateway.java

```diff
@@ -127,12 +130,8 @@ private ClusterMember mapMember(Member member) {
 		return cm;
 	}
 
-	private @Nonnull Authentication getAuthentication() {
-		Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
-		if(authentication == null) {
-			throw new AuthenticationServiceException("no Authentication object found in SecurityContext"); //This should technically not be possible but...
-		}
-		return authentication;
+	private @Nonnull String getAuthentication() {
+		return jwtGenerator.create();
```

Comment:
Kan je hier echt niets met de SecurityContextHolder icm Jwt? (in het kader van het performance issue).

Reply:
Hoe bedoel je?

Reply:
Ik begreep van de week tijdens de daily dat het iedere keer aanmaken van het token 'duur' was, ging dat dan niet hierom?